doc/widgets/labeling.md

```diff
@@ -0,0 +1,21 @@
+# Peak Labeling
+
+The peak labeling menu enables allows for individual or automatic peak labeling.
+This processes is achieved through the use of the Scipy find peaks toolkit.
+Using this, it is possible to find find peaks and use adjustable parameters 
+to adjust what is labeled as a peak even when plotting multiple spectra with
+overlapping peak positions. This process is limited by the clarity of data.
+As such, while it can preform on noisy datasets, it will yield clear results
+after data has been filtered. Added peaks can be remvoed by double right clicking. 
```

Comment:
Double-clicking is not something that can be easily discovered. Possible solutions, ideally both:
1. Right click on the line and the delete option in the submenu. 
2. An option to select the line by clicking, this selection should have a visual indicator and them pressing delete or backspace for removal.